### PR TITLE
Use new variable for the flash time to avoid old customization

### PR DESCRIPTION
**Pull request by matsl**
Hi Bob, When preparing this idea I saw your change with using the old variable and calculate the seconds to sleep. Might be a better approach but providing this anyway as food for thought.

### CODE REVIEW

**Comment by rswgnu** [comment]
Thanks.  We will leave that pending and revisit across time.

-- Bob

**Review by rswgnu — APPROVED** [review]

**Comment by matsl** [comment]
Added ChangeLog  to complete this PR.

File: hui-em-but.el [hunk 1]
```diff
@@ -44,10 +44,16 @@
   :group 'hyperbole-buttons)
 
 (defcustom hproperty:but-flash-time 1000
-  "*Emacs button flash delay."
+  "*Emacs button flash delay. Use of this variable is removed. Use
+`hproperty:but-flash-time-seconds' instead for controlling the flash time."
   :type '(integer :match (lambda (_widget value) (and (integerp value) (> value 0))))
```
Comment (rswgnu) [inline hui-em-but.el:49]: Look up deprecate variable in the Elisp manual for how we should do this.

**Review by rswgnu — APPROVED** [review]

File: hui-em-but.el (hunk 1 above)
Comment (matsl) [reply hui-em-but.el:48]: Uses now `make-obsolete-variable` instead.

**Review by rswgnu — APPROVED** [review]

**Pull request merged**